[DOCS] Alert details flyout & alerts histogram field selection (#1640)
elastic/security-docs: docs/detections/alerts-ui-manage.asciidoc

* Filter for a specific rule in the KQL bar (for example,
`kibana.alert.rule.name :"SSH (Secure Shell) from the Internet"`).

+
NOTE: KQL autocomplete for `.alerts-security.alerts-*` indices is available on the
*Alerts* and *Rule details* pages, and in Timeline when either `All` or
`Detection alerts` is selected.
+

* Use the date and time filter to define a specific time range. By default, this filter is set to search the last 24 hours.

+

> TIP: Use the Quick select menu to choose commonly or frequently used options.

Reviewer: Would wrap the UI menu option in bold.

*Alerts* and *Rule details* pages, and in Timeline when either `All` or
`Detection alerts` is selected.

[role="screenshot"]
image::images/rule-query.png[Shows how to search for a specific rule]

* Use the date and time filter to define a specific time range. By default, this filter is set to search the last 24 hours.

+
TIP: Use the Quick select menu to choose commonly or frequently used options.
+

[role="screenshot"]
image::images/update-timespan.gif[Shows how to update the time filter]

* View detection alerts generated by a specific rule. Go to *Detect* -> *Rules*, then select a rule name in the All rules table. The rule details page displays a comprehensive view of the rule's settings, and the Alerts table under the Trend histogram displays the alerts associated with the rule, including alerts from any previous or deleted revision of that rule.

[role="screenshot"]
image::images/rule-details.gif[Shows how to access alerts associated with a rule]

> * Group alerts by specific parameters and display individual alert counts by using the *Stack by* drop-down in the *Count* table. You can select or enter any aggregatable ECS field that is available on your alerts. For example, if you select `kibana.alert.rule.name`, the table displays the number of alerts generated by each rule.

Reviewer 2: ```suggestion
* Group alerts by specific parameters and display individual alert counts by using the *Stack by* drop-down in the *Count* table. You can select or enter any aggregatable ECS field that is available on your alerts. For example, if you select `kibana.alert.rule.name`, the table groups alerts by that field.
```

Author: While it does group the alert count by the field, I think I prefer the original because it describes what the user can expect to see, as opposed to how the table is aggregating alerts.

+
NOTE: The *Count* table resets to its default state if you navigate away from the Alerts page.
+

[role="screenshot"]
image::images/count-table.gif[Shows how to use the Stack by drop-down]

> * Visualize alerts by specific parameters by using the *Stack by* drop-down in the *Trend* histogram. You can select or enter any aggregatable ECS field that is available on your alerts. For example, if you select `kibana.alert.rule.name`, the histogram displays the total alert counts by rule name.

Reviewer 2: ```suggestion
* Visualize alerts by specific parameters by using the *Stack by* drop-down in the *Trend* histogram. You can select or enter any aggregatable ECS field that is available on your alerts. For example, if you select `kibana.alert.rule.name`, the histogram groups alerts by that field.

```

Reviewer 3: I think the original works here. Maybe only change could be `for each rule name` instead of `by rule name`, but same thing in my mind.. 🤷🏾

[discrete]
*Overview*

The *Overview* tab provides an overview of the alert and shows relevant threat intelligence details. Use this information to understand what generated the alert so you can appropriately triage and resolve it.

> The *Overview* tab has three main sections: the summary alert summary, *Highlighted Fields*, and *Enriched data*. The alert summary and *Highlighted Fields* sections are standard for every alert. The *Enriched data* section only displays for alerts with intelligence data.

Reviewer 2: ```suggestion
The *Overview* tab has three main sections: the alert summary, *Highlighted Fields*, and *Enriched data*. The alert summary and *Highlighted Fields* sections are standard for every alert. The *Enriched data* section only displays for alerts with intelligence data.
```

Reviewer 3: My only thing here is while the *Highlighted Fields* section is present for every alert, the fields aren't standard between alerts as the fields shown are based on the rule type and available data. Unless I missed something and something changed. Maybe:

" The alert summary section is standard for every alert. The Highlighted Fields section will show data certain fields based on rule type as well as the underlying data. ...."

@janmonschke would be able to help us clarify for sure

Reviewer 3: Ignore me, missed the comment below: `The *Highlighted Fields* section is also provided for all alerts and is customized to only show the most relevant fields for the alert type. Use this section to inform your triage efforts as you investigate the alert.` 

Thanks!

[role="screenshot"]
image::images/alert-details-flyout.png[Shows the alert details flyout]

> The alert summary section provides general alert details such as the alert's status, the severity of the alert, the alert's calculated risk score, and a link to the rule that produced the alert. These details are provided for all alerts regardless of the alert type, and they offer an overview of important alert details.

Reviewer 2: ```suggestion
The alert summary section provides general alert details such as the alert's status, the severity of the alert, the alert's calculated risk score, and a link to the rule that produced the alert. These details are provided for all alerts regardless of the alert type.
```
I feel like you already made this clear

> The alert's reason statement is under the alert summary. The reason statement describes what generated the alert and provides general alert details. You can use this information to understand the alert's origin and determine if the alert is relevant to your investigation.

Reviewer 2: Is it a reason section, or a description? You could be right, and I know it used to say reason, but to me they generally read like descriptions.

Author: I kept the original terminology because the statement matches the text shown in the Reason column when you view the alert from the Alert table. That said, I'm always open to representing it in a clearer way if "reason statement" is too ambiguous. @michaelolo24 what are your thoughts on continuing to use "reason statement" to describe the statement under the alert's main details? Should we continue to frame the text that way to align it with the Reason column in the Alerts table or change it to something else, like "alert description"?

Reviewer 3: We should continue with the reason statement because it is that field in the table `kibana.alert.reason`, but also in the long run, we want to enhance this statement so keeping this naming will align with that
